Application/Source/Detail/View/DetailViewModel.swift

```diff
@@ -25,6 +25,21 @@ class DetailViewModel: ViewModel, SelectionPresentingViewModel {
         return .empty()
     }
 
+    let content: BehaviorRelay<[Food]> = BehaviorRelay(value: [.beans, .greens, .potatoes, .tomatoes])
+
+    let contentListSeparator: String = ", "
```

Comment:
Is this only declared here so that it can also be used in the unit test? I do think it's acceptable to just redeclare this separator in the unit test. In this case I think it cleans up the code enough to be worth just declaring it in line both in this class and in the unit test.

On top of cleaning up some of the code below, this exposes a public property to consumers of this class that shouldn't really be exposed in an ideal world (without considering the unit test).

If you would like to abstract it though, perhaps this should probably be a `static let` so that you don't need to reference self in your mapping block below..